com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,75 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+#rename-appdata-file: 'bitwig-studio.appdata.xml'
+finish-args:
+  # TODO: we need MIDI, Audio, DRI, and realtime performances
+  - --device=all
+  - --nodevice=kvm
```

Comment:
What's the story behind that one? It's unusual.

Reply:
We want to have access to midi and usb devices.

Reply:
@abique I guess the thing in question is `--nodevice=kvm`.

Reply:
we have --device=all, so I now need to exclude kvm right?

Reply:
No, why? IMO `/dev/kvm` isn't really sensitive. `/dev/input` is, but we can't exclude it anyway.

Reply:
OK, I can remove kvm